### DISCUSSION TITLE
equivalence check fails during CTS

### DISCUSSION THREAD

**Discussion by mercury-5** [The-OpenROAD-Project/OpenROAD-flow-scripts#1664] [General]
Error: cts.tcl, 110 couldn't execute "eqy": no such file or directory
Command exited with non-zero status 1
Elapsed time: 0:03.23[h:]min:sec. CPU time: user 3.13 sys 0.05 (98%). Peak memory: 126656KB.
make[1]: *** [Makefile:691: do-4_1_cts] Error 1
make: *** [Makefile:691: results/sky130hd/vending_machine/base/4_1_cts.odb] Error 2

**Comment by maliberty** [comment]
You need to install eqy

**Reply by mercury-5** [reply]
How to install it

**Reply by maliberty** [reply]
@vvbandeira is this in the docs somewhere?

**Reply by vijayank88** [reply]
@mercury-5 try the solution from here: https://github.com/The-OpenROAD-Project/OpenROAD/issues/4035#issuecomment-1734753815
